f/assignGear/f_assignGear_ctrg.sqf

```diff
@@ -233,8 +233,6 @@ _APmine2 = "APERSMine_Range_Mag";
 // Define classes. This defines which gear class gets which uniform
 // "medium" vests are used for all classes if they are not assigned a specific uniform
 
-_light = [];
-_heavy =  ["eng","engm"];
```

Comment:
I decided to remove these "classes", since there is no logic/code to set and apply clothes/items for these "classes", in contrast to the others (diver, pilot, crew.
See a few lines below and in `f_assignGear_clothes`.

In my opinion it's confusing to have these arrays that do nothing.
It would be better to recreate them once they're actually used instead of leaving them in as dead code.

Reply:
Just comment them out instead, maybe? Could add a note that they don't do anything by default.